demo/apisix/conf/apisix.yaml

```diff
@@ -0,0 +1,18 @@
+routes:
+  -
+    id: httpbin    #For test, TODO remove
```

Comment:
Question: Does the hyphen need to stand alone in 1 line? Looks weird to me.

Reply:
I copied the format in the document example 
https://apisix.apache.org/docs/apisix/3.1/stand-alone/

Reply:
Just test 
It works with bellow format 
```yaml
routes:
  - id: test   
    uri: /ping
```

I think it is just a format convention, no big deal, we can keep the format similar to apisix document